Description: Remove bytestring-builder from Build-depends
 This package is a no-op for Debian (and so, we do not ship it).
Author: Joachim Breitner <nomeata@debian.org>
Forwarded: not-needed, debian specific
--- a/conduit-extra.cabal
+++ b/conduit-extra.cabal
@@ -73,7 +73,6 @@
 
                    , async
                    , attoparsec
-                   , bytestring-builder
                    , bytestring
                    , exceptions
                    , process
@@ -108,7 +107,6 @@
                   , conduit-extra
                   , gauge
                   , bytestring
-                  , bytestring-builder
                   , transformers
     main-is:        blaze.hs
     ghc-options:    -Wall -O2 -rtsopts
